src/index.rs

```diff
@@ -49,6 +51,14 @@ pub struct IndexEntry {
     #[serde(default)]
     pub mtime: i64,
 
+    /// Discretionary Access Control permissions (such as read/write/execute on unix)
```

Comment:
Could you please also mention these in https://github.com/sourcefrog/conserve/blob/main/doc/format.md

Reply:
done